front/app/containers/IdeasNewPage/WithJSONForm/Heading.tsx

```diff
@@ -102,9 +102,10 @@ export const Heading = ({
                     data-cy="e2e-edit-survey-link"
                     icon="edit"
                     linkTo={linkToSurveyBuilder}
-                    buttonStyle="text"
+                    buttonStyle="primary-inverse"
                     textDecorationHover="underline"
```

Comment:
Do we need some more padding between the edit button and the "X"? Just noticed the prototype has a bit more, but not sure if it's outdated?

![image](https://user-images.githubusercontent.com/33987955/204802211-c8083939-5382-46db-bffc-980d06b6c132.png)

Reply:
Added a little more margin